src/main/java/SeungYoon.java

```diff
@@ -0,0 +1,41 @@
+import static data.Constants.*;
+
+/*
+ * Coding standard is defined by IDE
+ * lowerCamelCase for variables and object instances
+ * UpperCamelCase for Class names
+ * ALL_CAPS_WITH_UNDERSCORE for constants
+ * Print messages encapsulated in private static methods for readability
```

Comment:
These comments seem as if they are written for your reference and not for the reader.